src/ImageRunModal.scss

```diff
@@ -1,17 +1,18 @@
-@import '@patternfly/patternfly/layouts/Grid/grid.css';
+.dynamic-form-group {
+    .pf-c-empty-state {
+        padding: 0;
+    }
```

Comment:
When I was working on the design I used:
```
 <EmptyState variant='xs'>
```

Reply:
Done